.eslintrc

```diff
@@ -1,7 +1,6 @@
 {
   "extends": ["react-app", "plugin:react-hooks/recommended"],
   "rules": {
-    "react/require-default-props": 2,
```

Comment:
Removed this, as it's forcing us to add `null` values in many places where `undefined` is perfectly fine. This will also allow us to default values directly in the initial FC prop spread without linting warnings like

```jsx
const MyComponent = ({ 
  someVal = 1, 
  someFunc = () => true 
}) => {
  ...
}
```